debian/control

```diff
@@ -7,7 +7,7 @@ Vcs-Browser: https://github.com/ceph/ceph
 Maintainer: Ceph Maintainers <ceph-maintainers@lists.ceph.com>
 Uploaders: Ken Dreyer <kdreyer@redhat.com>,
            Alfredo Deza <adeza@redhat.com>,
-Build-Depends: cmake (>= 3.5),
+Build-Depends: cmake (>= 3.10.2),
```

Comment:
could you put the rationales of this change in the commit message?

Reply:
Hey, it's done.

Reply:
please update the commit message, and repush.

Reply:
also, we've dropped the support of xenial in the sense that we don't build on it anymore.